official/recommendation/data_pipeline.py

```diff
@@ -382,7 +388,8 @@ def __init__(self,
 
     self._shuffle_with_forkpool = not stream_files
     if stream_files:
-      self._shard_root = tempfile.mkdtemp(prefix="ncf_")
+      self._shard_root = (
```

Comment:
This can be rewritten:
`self._shard_root = epoch_dir or tempfile.mkdtemp(prefix="ncf_")`